changelog.d/dump-il.fixed

```diff
@@ -0,0 +1 @@
+Changed it such that `dump-il` prints out all the top-level IL statements of a program, rather than individually printing out all of the statements in each function.
```

Comment:
In general we do not add stuff useful only for us internally in the changelog (e.g., refactoring, improving the build system, or stuff related to the semgrep-core command-line program since users are not supposed to use semgrep-core directly but use semgrep instead.